Code/Tools/SceneAPI/SceneCore/Utilities/SceneGraphSelector.cpp

```diff
@@ -40,18 +42,31 @@ namespace AZ
                 return object && object->RTTI_IsTypeOf(DataTypes::IMeshData::TYPEINFO_Uuid());
             }
 
-            Containers::SceneGraph::NodeIndex SceneGraphSelector::RemapToOptimizedMesh(const Containers::SceneGraph& graph, const Containers::SceneGraph::NodeIndex& index)
+            Containers::SceneGraph::NodeIndex SceneGraphSelector::RemapToOptimizedMesh(
+                const Containers::SceneGraph& graph,
+                const Containers::SceneGraph::NodeIndex& index,
+                const DataTypes::ISceneNodeGroup& sceneNodeGroup)
             {
                 const auto& nodeName = graph.GetNodeName(index);
-                const AZStd::string optimizedName = AZStd::string(nodeName.GetPath(), nodeName.GetPathLength()).append(OptimizedMeshSuffix);
+
+                const AZStd::string optimizedName = AZStd::string(nodeName.GetPath(), nodeName.GetPathLength())
+                                                        .append("_")
+                                                        .append(sceneNodeGroup.GetName())
+                                                        .append(SceneAPI::Utilities::OptimizedMeshSuffix);
```

Comment:
this can be replaced with `AZStd::string::format`

Reply:
Done.